### ISSUE TITLE
mbstring.func_overload can prevent strlen from returning a byte count.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/minify#132]
```
Mark Reidenbach e-mailed this issue and the attached patches.

What steps will reproduce the problem?
1. In php.ini, set mbstring.func_overload to x such that x & 2 = 1. 
2. Minify a script with multibyte characters

JSMin will incorrectly count the bytes of the script (potentially 
introducing errors) and Minify will set an incorrect Content-Length 
header, breaking client display.
```

---

Original issue reported on code.google.com by `mrclay....@gmail.com` on 25 Aug 2009 at 9:43

Attachments:
- [JSMin-mbstring.func_overload.diff](https://storage.googleapis.com/google-code-attachments/minify/issue-132/comment-0/JSMin-mbstring.func_overload.diff)
- [Minify-mbstring.func_overload.diff](https://storage.googleapis.com/google-code-attachments/minify/issue-132/comment-0/Minify-mbstring.func_overload.diff)

**Comment by GoogleCodeExporter** [comment]
```
Original jsmin-php patch for future ref
http://github.com/rgrove/jsmin-php/commit/f89f32db50e943515c7361ebcfb6a2cacf7dc9
1a
```

---

Original comment by `mrclay....@gmail.com` on 21 Dec 2009 at 5:14

**Comment by GoogleCodeExporter** [comment]
```
Mark was using these settings:

default_charset = "UTF-8"
mbstring.internal_encoding = UTF-8
mbstring.func_overload = 7

and minifying the attached files
```

---

Original comment by `mrclay....@gmail.com` on 9 May 2010 at 3:49

Attachments:
- [mbstring_test_files.zip](https://storage.googleapis.com/google-code-attachments/minify/issue-132/comment-2/mbstring_test_files.zip)

**Comment by GoogleCodeExporter** [comment]
```
These issues should be fixed in 2.1.4
```

---

Original comment by `mrclay....@gmail.com` on 30 Sep 2010 at 4:36
- Changed state: **Fixed**
